+++
authors = ["Zachary A King", "Andreas Dr√§ger", "Ali Ebrahim", "Nikolaus Sonnenschein", "Nathan E Lewis", "Bernhard O Palsson"]
title = "Escher: A Web Application for Building, Sharing, and Embedding Data-Rich Visualizations of Biological Pathways."
journal = "PLoS computational biology"
what = "article"
doi = "10.1371/journal.pcbi.1004321"
pubmed = "26313928"
date = "2015-08-28"
keywords = []
+++

Escher is a web application for visualizing data on biological pathways. Three key features make Escher a uniquely effective tool for pathway visualization. First, users can rapidly design new pathway maps. Escher provides pathway suggestions based on user data and genome-scale models, so users can draw pathways in a semi-automated way. Second, users can visualize data related to genes or proteins on the associated reactions and pathways, using rules that define which enzymes catalyze each reaction. Thus, users can identify trends in common genomic data types (e.g. RNA-Seq, proteomics, ChIP)--in conjunction with metabolite- and reaction-oriented data types (e.g. metabolomics, fluxomics). Third, Escher harnesses the strengths of web technologies (SVG, D3, developer tools) so that visualizations can be rapidly adapted, extended, shared, and embedded. This paper provides examples of each of these features and explains how the development approach used for Escher can be used to guide the development of future visualization tools.