+++
authors = ["Ali Ebrahim", "Elizabeth Brunk", "Justin Tan", "Edward J O'Brien", "Donghyuk Kim", "Richard Szubin", "Joshua A Lerman", "Anna Lechner", "Anand Sastry", "Aarash Bordbar", "Adam M Feist", "Bernhard O Palsson"]
title = "Multi-omic data integration enables discovery of hidden biological regularities."
journal = "Nature communications"
what = "article"
doi = "10.1038/ncomms13091"
pubmed = "27782110"
date = "2016-10-26"
keywords = []
+++

Rapid growth in size and complexity of biological data sets has led to the 'Big Data to Knowledge' challenge. We develop advanced data integration methods for multi-level analysis of genomic, transcriptomic, ribosomal profiling, proteomic and fluxomic data. First, we show that pairwise integration of primary omics data reveals regularities that tie cellular processes together in Escherichia coli: the number of protein molecules made per mRNA transcript and the number of ribosomes required per translated protein molecule. Second, we show that genome-scale models, based on genomic and bibliomic data, enable quantitative synchronization of disparate data types. Integrating omics data with models enabled the discovery of two novel regularities: condition invariant in vivo turnover rates of enzymes and the correlation of protein structural motifs and translational pausing. These regularities can be formally represented in a computable format allowing for coherent interpretation and prediction of fitness and selection that underlies cellular physiology.