+++
authors = ["Daniel A Cuevas", "Janaka Edirisinghe", "Chris S Henry", "Ross Overbeek", "Taylor G O'Connell", "Robert A Edwards"]
title = "From DNA to FBA: How to Build Your Own Genome-Scale Metabolic Model."
journal = "Frontiers in microbiology"
what = "article"
doi = "10.3389/fmicb.2016.00907"
pubmed = "27379044"
date = "2016-06-17"
keywords = ["flux-balance analysis", "genome annotation", "in silico modeling", "metabolic modeling", "metabolic reconstruction", "model SEED"]
+++

Microbiological studies are increasingly relying on in silico methods to perform exploration and rapid analysis of genomic data, and functional genomics studies are supplemented by the new perspectives that genome-scale metabolic models offer. A mathematical model consisting of a microbe's entire metabolic map can be rapidly determined from whole-genome sequencing and annotating the genomic material encoded in its DNA. Flux-balance analysis (FBA), a linear programming technique that uses metabolic models to predict the phenotypic responses imposed by environmental elements and factors, is the leading method to simulate and manipulate cellular growth in silico. However, the process of creating an accurate model to use in FBA consists of a series of steps involving a multitude of connections between bioinformatics databases, enzyme resources, and metabolic pathways. We present the methodology and procedure to obtain a metabolic model using PyFBA, an extensible Python-based open-source software package aimed to provide a platform where functional annotations are used to build metabolic models (http://linsalrob.github.io/PyFBA). Backed by the Model SEED biochemistry database, PyFBA contains methods to reconstruct a microbe's metabolic map, run FBA upon different media conditions, and gap-fill its metabolism. The extensibility of PyFBA facilitates novel techniques in creating accurate genome-scale metabolic models.