+++
authors = ["Sergio Bordel"]
title = "Constraint based modeling of metabolism allows finding metabolic cancer hallmarks and identifying personalized therapeutic windows."
journal = "Oncotarget"
what = "article"
doi = "10.18632/oncotarget.24805"
pubmed = "29731977"
date = "2018-04-13"
keywords = ["RNA-seq", "flux balance analysis", "metabolic models", "metabolism", "therapeutic windows"]
+++

In order to choose optimal personalized anticancer treatments, transcriptomic data should be analyzed within the frame of biological networks. The best known human biological network (in terms of the interactions between its different components) is metabolism. Cancer cells have been known to have specific metabolic features for a long time and currently there is a growing interest in characterizing new cancer specific metabolic hallmarks. In this article it is presented a method to find personalized therapeutic windows using RNA-seq data and Genome Scale Metabolic Models. This method is implemented in the python library, pyTARG. Our predictions showed that the most anticancer selective (affecting 27 out of 34 considered cancer cell lines and only 1 out of 6 healthy mesenchymal stem cell lines) single metabolic reactions are those involved in cholesterol biosynthesis. Excluding cholesterol biosynthesis, all the considered cell lines can be selectively affected by targeting different combinations (from 1 to 5 reactions) of only 18 metabolic reactions, which suggests that a small subset of drugs or siRNAs combined in patient specific manners could be at the core of metabolism based personalized treatments.