+++
authors = ["Yara Seif", "Jonathan M Monk", "Nathan Mih", "Hannah Tsunemoto", "Saugat Poudel", "Cristal Zuniga", "Jared Broddrick", "Karsten Zengler", "Bernhard O Palsson"]
title = "A computational knowledge-base elucidates the response of Staphylococcus aureus to different media types."
journal = "PLoS computational biology"
what = "article"
doi = "10.1371/journal.pcbi.1006644"
pubmed = "30625152"
date = "2019-01-09"
keywords = []
+++

S. aureus is classified as a serious threat pathogen and is a priority that guides the discovery and development of new antibiotics. Despite growing knowledge of S. aureus metabolic capabilities, our understanding of its systems-level responses to different media types remains incomplete. Here, we develop a manually reconstructed genome-scale model (GEM-PRO) of metabolism with 3D protein structures for S. aureus USA300 str. JE2 containing 854 genes, 1,440 reactions, 1,327 metabolites and 673 3-dimensional protein structures. Computations were in 85% agreement with gene essentiality data from random barcode transposon site sequencing (RB-TnSeq) and 68% agreement with experimental physiological data. Comparisons of computational predictions with experimental observations highlight: 1) cases of non-essential biomass precursors; 2) metabolic genes subject to transcriptional regulation involved in Staphyloxanthin biosynthesis; 3) the essentiality of purine and amino acid biosynthesis in synthetic physiological media; and 4) a switch to aerobic fermentation upon exposure to extracellular glucose elucidated as a result of integrating time-course of quantitative exo-metabolomics data. An up-to-date GEM-PRO thus serves as a knowledge-based platform to elucidate S. aureus' metabolic response to its environment.